css/properties/will-change.json

```diff
@@ -15,40 +15,14 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": [
-              {
-                "version_added": "36",
-                "notes": "Before Firefox 43, Firefox supported <code>will-change: will-change</code>, which is invalid by the specification."
-              },
-              {
-                "version_added": "31",
-                "version_removed": "36",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.will-change.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "36",
-                "notes": "Before Firefox 43, Firefox supported <code>will-change: will-change</code>, which is invalid by the specification."
-              },
-              {
-                "version_added": "31",
-                "version_removed": "36",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.will-change.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox": {
+              "version_added": "36",
+              "notes": "Before Firefox 43, Firefox supported <code>will-change: will-change</code>, which is invalid by the specification."
```

Comment:
I don't think there's any value in keeping this note. @ddbeck?

Reply:
I agree. This is very obscure. Based on [the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1195884) (which is the only evidence I can find for `will-change: will-change`, apart from old translations of MDN pages), I'm not convinced this ever mattered to anyone who wasn't a spec author or implementer.